parachains/integration-tests/statemine/xcm/3_hrmp-open-channels.yml

```diff
@@ -170,245 +77,76 @@ tests:
           - name: Penpal Parachain sends a request to the Relay Chain to open a channel with the Assets Parachain
```

Comment:
Seems like the context has changed here. We are no longer sending the request from Penpal, but rather we call an extrinsic on the relay chain to open the channel. I guess we should then change the name of this test to something that reflects this scenario better.

Reply:
Got it fixed in `parachains-integration-tests` repo.